modules/search-service-impl/src/main/java/org/opencastproject/search/impl/SearchServiceImpl.java

```diff
@@ -361,10 +360,6 @@ public void addSynchronously(MediaPackage mediaPackage) throws SearchException,
           IllegalArgumentException, UnauthorizedException {
     User currentUser = securityService.getUser();
```

Comment:
The variables currentUser and orgAdminRole are not used after you remove the useless check.

Reply:
good catch. removed.